recipes-bsp/bootfiles/rpi-cmdline.bb

```diff
@@ -57,6 +57,7 @@ CMDLINE = " \
     ${CMDLINE_PITFT} \
     ${CMDLINE_DEBUG} \
     ${CMDLINE_RNDIS} \
+    net.ifnames=0 \
```

Comment:
Lets make it possible to override it if someone want to enable renaming. So do something like

`CMDLINE_IFNAMES ?= "net.ifnames=0"`

then add following to `CMDLINE`

`${CMDLINE_IFNAMES}`

Reply:
Thank you for your suggestion.

Reply:
Please squash both changes into single commit.